### PR TITLE
chore: use cross-env in npm scripts

### LINKED ISSUE #4667: Use cross-env in NPM scripts
**Describe the bug**
When building on a Windows computer, npm scripts like `dev`, `start` and `webpack:dev:client` fail because windows shell environments like CMD and Powershell, do not recognize `NODE_ENV=development` as a way to set environment variables.

**To Reproduce**

1. Clone simorgh on a windows computer
2. Run `npm install`
3. Run `npm run dev`

**Expected behavior**
NPM scripts should not fail due to environment variable set syntax

**Screenshots**

image

**Desktop (please complete the following information):**

- OS: [e.g. Windows 10]

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by mykeels**
Resolves #4667

**Overall change:** _Use cross-env to prevent npm scripts breaking on Windows, due to differences in the expected syntax for setting env variables._

**Code changes:**

- _Add cross-env package, and use in npm scripts._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,6 +89,7 @@
     "@loadable/webpack-plugin": "^5.7.1",
     "compression": "^1.7.4",
     "core-js": "^3.4.1",
+    "cross-env": "^6.0.3",
```
Comment (AlistairGempf) [inline package.json]: Shouldn't it be installed as a dev dependency?

Comment (andrewscfc) [reply package.json]: ➕ above

**Review by andrewscfc — COMMENTED** [review]
Just change to `cross-env` to dev dependency and happy to approve

File: package.json (hunk 1 above)
Comment (mykeels) [reply package.json]: Done!

**Review by andrewscfc — APPROVED** [review]

**Review by AlistairGempf — APPROVED** [review]
Looks good

**Review by ryanmccombe — APPROVED** [review]
Looks good 👍 

Does this not need applied to the `start` and `webpack:dev:client` scripts too?

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**